runner/engine.go

```diff
@@ -431,31 +431,37 @@ func (e *Engine) building() error {
 }
 
```

Comment:
we need to add a new flag to support this, and disable this behavior by default

Reply:
ok